QtCollider/widgets/QcWebView.cpp

```diff
@@ -35,13 +34,15 @@
 #    include <QStyle>
 #    include <QWebEngineCallback>
 
+#    ifndef SCIDE_NO_QWIDGETFACTORY
+#        include "../QcWidgetFactory.h"
 QC_DECLARE_QWIDGET_FACTORY(WebView);
+#    endif
```

Comment:
so, instead of this macro is it possible to just move this bit into a new cpp file? that would be a lot cleaner.